url_utils/plugin.spec.yaml

```diff
@@ -0,0 +1,67 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: url_utils
+title: URL Utilities
+description: Encode and decode URLs
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+cloud_ready: true
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/url_utils
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- url
+- encoder
+- decoder
+- utilities
+hub_tags:
+  use_cases: [data_utility]
+  keywords: [url, encoder, decoder, utilities]
+  features: []
+actions:
+  encode:
+    title: Encoder
+    description: Encode URLs
+    input:
+      url:
```

Comment:
Needs a "title" property here, otherwise the input will show up as "Url" in ICON. 

title: "URL"